src/Tribe/Repositories/Attendee/RSVP.php

```diff
@@ -104,4 +105,126 @@ public function checked_in_keys() {
 		return $this->limit_list( $this->key_name, parent::checked_in_keys() );
 	}
 
+	/**
```

Comment:
* :no_entry_sign: Doc comment for parameter "$attendee" missing
* :no_entry_sign: Doc comment for parameter "$ticket" missing
* :no_entry_sign: Doc comment for parameter "$attendee_data" missing

<sub>hash: bd03078477f0b36657b5197976fa3a40</sub>